src/client/client.go

```diff
@@ -192,5 +197,28 @@ func setupLogs(client *Client, verbose bool) {
 
 func responseToLog(res *resty.Response) string {
 	req := res.Request
-	return fmt.Sprintf("%s %s | %d | %s", req.Method, req.URL, res.StatusCode(), res.Time())
+	return fmt.Sprintf("%s %s | %d | %s", req.Method, urlForLog(req), res.StatusCode(), res.Time())
+}
+
+func urlForLog(request *resty.Request) string {
+	url := request.URL
+
+	// No response -> url contains placeholders
+	if request.RawRequest == nil {
+		pathParams := request.Context().Value(contextKey("pathParams")).(map[string]string)
```

Comment:
No len s tym nastal jeden problem, ... ze placeholders z url, napr `{branchId}` sa nahradzuju az tesne pred odoslanim requestu. A tak som v logoch nevidel, o ktory request sa jedna.

Preto su to musim pred zalogovanim nahradit sam.
Iny sposob ako tu dostat `pathParams` je -> cez `context` .... kedze `pathParams` su v `resty` kniznici private, ... tak si ich musim tiez ulozit a nastavit do `context`.